src/admin/redux/modules/letter.js

```diff
@@ -36,8 +36,12 @@ const getAddress = (
   , postcode
   }
 ) =>
-  [ `${title} ${first_name[0] || ''} ${last_name}`, address1, address2,
-    address3, address4, county, postcode ]
+  [ `${title} ${(first_name && first_name[0]) || ''} ${last_name}`
```

Comment:
Of course, of course. Should have remembered `format_date` from last week. I am beginning to see how these FP concepts can come in very handy.